src/Core/src/Platform/Android/LabelExtensions.cs

```diff
@@ -20,7 +21,18 @@ public static void UpdateTextColor(this TextView textView, ILabel label,  Maui.C
 			else
 			{
 				textView.SetTextColor(textColor.ToNative());
-			}				
+			}
+		}
+
+		public static void UpdateFont(this TextView textView, ILabel label, IFontManager fontManager)
```

Comment:
I decided to pass in the required font manager here to keep things clean down here and let the handlers be a more interop-y thing.